File: Python/1285.py
inStr = input()
num = ''
tmpList = []

for i in inStr:
    if i not in ['+', '-', '*', '/', '=']:
        num += i
    else:
        tmpList.append(int(num))
        tmpList.append(i)
        num = ''

sum = tmpList[0]

for j in range(len(tmpList)):
    if tmpList[j] == "+":
        sum += tmpList[j + 1]
    elif tmpList[j] == "-":
        sum -= tmpList[j + 1]
    elif tmpList[j] == "*":
        sum *= tmpList[j + 1]
    elif tmpList[j] == "/":
        sum = int(sum / tmpList[j + 1])

print(int(sum))

File: C++/1285.cpp
#include <iostream>

using namespace std;

int main(){
 
    char ch;
    int num;
    int sum = 0;
    cin >> num;
    sum = num;
    for(int i = 0;;i++){
        cin >> ch;
        
        if(ch == '+'){
            cin >> num;
            sum += num;
        }
        
        else if(ch == '-'){
            cin >> num;
            sum -= num;
		}
        else if(ch == '/'){
            cin >> num;
            sum /= num;
		}
        else if(ch == '*'){
            cin >> num;
            sum *= num;
		}
        else if(ch == '='){
            cout << sum;
            break;
        }
    }
}